### DISCUSSION TITLE
Add support for tabs, so that I could multi-task between repos, PRs, .. in the iOS app directly

### DISCUSSION THREAD

**Discussion by IdanAdar** [community/community#129175] [Mobile]
### Select Topic Area

Product Feedback

### Body

Add support for tabs, so that I could multi-task between repos, PRs, .. in the iOS app directly

**Comment by esahmed21** [comment]
Hi @IdanAdar 👋 

Thank you for your suggestion! We appreciate your input and it's great idea, I will forward it to our team for further evaluation and consideration.

Thank you for reaching out!

**Reply by deinauno17** [reply]
Successful treatment surprisely patiently in words modern access information very thoughtful in learning especialized the privacy policy secure and helping carefuly the situation difficultly coming to solving the problem ofhe other it was makes sovereinty that the users make there heartmotion good feel and survivor teams come true
It's happily waiting a moment impossible come in

**Reply by IdanAdar** [reply]
> Hi @IdanAdar 👋 
> 
> 
> 
> Thank you for your suggestion! We appreciate your input and it's great idea, I will forward it to our team for further evaluation and consideration.
> 
> 
> 
> Thank you for reaching out!

Thanks! Is there an internal ticket for this request? Has any progress been made?
